puppetdb/Dockerfile

```diff
@@ -112,11 +112,15 @@ RUN dpkg -i /${PUPPET_DEB} && \
     apt autoremove && \
     rm -rf /var/lib/apt/lists/* && \
     mkdir -p "$LOGDIR" && \
+    chown puppetdb:puppetdb "$LOGDIR" && \
 # We want to use the HOCON database.conf and config.conf files, so get rid
 # of the packaged files
     rm -f /etc/puppetlabs/puppetdb/conf.d/database.ini && \
     rm -f /etc/puppetlabs/puppetdb/conf.d/config.ini
 
+RUN apt update && \
```

Comment:
plsease add this to L73 or L111 to not create a new layer which would consume more space

Reply:
Done